Corona-Warn-App/src/main/assets/terms_de.html

```diff
@@ -245,61 +284,66 @@ <h1>
+    Symptombeginn) bereitgestellten Daten und lässt keine Aussage über das
+    tatsächliche Vorliegen einer Infektion zu.
 </p>
 <p>
     <strong>
-        Die Benachrichtigung über eine Risiko-Begegnung kann unzutreffend sein.
-    </strong>
-    Die Risiko-Begegnung kann von Ihrem Smartphone beispielsweise zu einem
-    Zeitpunkt registriert worden sein, zu dem Sie sich nicht in der Nähe Ihres
-    Smartphones aufgehalten haben oder während eine andere Person Ihr Smartphone
-    verwendet hat. Die Risiko-Begegnung kann auch aufgrund bestehender Grenzen
-    bei der Kontaktmessung fälschlicherweise registriert worden sein (siehe
-    unten Ziffer 8).
-</p>
-<h1>
-    4. KEIN INFEKTIONSSCHUTZ DURCH DIE APP
-</h1>
-<p>
-    Die App dient der Unterbrechung von Infektionsketten.
-</p>
-<p>
-    • <strong>Die App schützt Sie nicht vor einer SARS-CoV-2-Infektion. </strong>
-</p>
-<p>
-    •
-    <strong>
-        Die App reduziert Ihr persönliches Infektionsrisiko nicht.
+        Die Information über eine Risiko-Begegnung kann unzutreffend sein.
     </strong>
+    Die Risiko-Begegnung kann vom COVID-19-Benachrichtigungssystem
+    beispielsweise zu einem Zeitpunkt aufgezeichnet worden sein, zu dem Sie
+    sich nicht in der Nähe Ihres Endgeräts aufgehalten haben oder während eine
+    andere Person Ihr Endgerät verwendet hat. Die Risiko-Begegnung kann auch
+    aufgrund bestehender Grenzen des COVID-19-Benachrichtigungssystems
+    fälschlicherweise registriert worden sein (siehe unten Ziffer 8).
 </p>
 <p>
-    •
-    <strong>
-        Sie können sich auch mit SARS-CoV-2 infizieren, ohne dass die App Sie
-        über die Risiko-Begegnung mit der Person benachrichtigt, die Sie
-        infiziert hat:
-    </strong>
+    4. KEIN INFEKTIONSSCHUTZ DURCH DIE APP
 </p>
 <p>
-    o Die App registriert nicht alle Ihre Begegnungen mit anderen Personen,
-    z.B. weil andere Personen die App nicht verwenden, Sie Ihr Smartphone nicht
-    immer bei sich tragen oder die App nicht immer in Betrieb haben oder weil
-    die Kontaktmessung gewissen Grenzen unterliegt (siehe unten Ziffer 8).
+    Die App dient der Unterbrechung von Infektionsketten.
 </p>
-<p>
-    o Die App informiert Sie nur über festgestellte Risiko-Begegnungen mit
-    infizierten Personen. Das setzt voraus, dass die App die Begegnung mit der
-    infizierten Person registriert hat <u>und</u> die infizierte Person eine
+<ul>
+    <li>
+        <strong>
+            Die App schützt Sie nicht vor einer SARS-CoV-2-Infektion.
+        </strong>
+    </li>
+    <li>
+        <strong>
+            Die App reduziert Ihr persönliches Infektionsrisiko nicht.
+        </strong>
+    </li>
+    <li>
+        <strong>
+            Sie können sich auch mit SARS-CoV-2 infizieren, ohne dass die App
+            Sie über die Risiko-Begegnung mit der Person informiert, die Sie
+            infiziert hat:
+        </strong>
+    </li>
+</ul>
```

Comment:
Is the md syntax correct here? The bulleted lists ends with the third bullet, but the following paragraphs belong to the third bullet.